PowerShellWorker.Common.props

```diff
@@ -1,9 +1,9 @@
 <Project>
   <PropertyGroup>
     <BuildNumber Condition=" '$(BuildNumber)' == '' ">1</BuildNumber>
-    <Version>3.0.$(BuildNumber)</Version>
+    <Version>4.0.$(BuildNumber)</Version>
     <PackageTags>PowerShell;AzureFunctions;language;Azure;</PackageTags>
-    <PackageLicenseUrl>https://github.com/Azure/azure-functions-powershell-worker/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageLicenseUrl>https://github.com/Azure/azure-functions-powershell-worker/blob/dev/LICENSE</PackageLicenseUrl>
```

Comment:
Not sure this should be or needs to be changed

Reply:
@kilasuit -- we no longer have a `master` branch, so when you click at the link above, it redirects you to the default branch, in this case `dev`.